index.html

```diff
@@ -851,6 +885,20 @@ <h3>
           <li>Return |buttons|.
           </li>
         </ol>
+        <p>
+          To <dfn data-lt="initializing the vibration actuator">initialize the
+          vibration actuator</dfn> for a gamepad, run the following steps:
+        </p>
+        <ol>
+          <li>If |gamepad:Gamepad| has a vibration actuator that can
```

Comment:
This seems overly specific... I know we might not change this in the future, but making it depend on "dual-rumble" exclusively doesn't seem great. This should probably just be that it it supports haptic effects.